libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Matrix/MatrixCells/MatrixCells.tsx

```diff
@@ -143,7 +143,14 @@ export class MatrixCells extends React.PureComponent<IMatrixCellsProps> {
               baseError = falseCount;
             } else {
               metricValue = value.metricValue;
```

Comment:
Is metricValue now unused?

Reply:
it is used in the FilterProps constructor below